### ISSUE TITLE
Error when plot_lollipop with the reshaped gsea results

### ISSUE THREAD

**Issue by barizona** [ELTEbioinformatics/mulea#8]
There is a segmentation error when plotting lollipop with the gsea reshaped results:
"Quitting from lines 480-487 [lollipop_plot_gsea] (README.Rmd)
Error in `geom_segment()`:
! Problem while computing aesthetics.
ℹ Error occurred in the 1st layer.
Caused by error:
! object 'eFDR' not found
Backtrace:
  1. rmarkdown::render(...)
  2. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
  3. knitr:::process_file(text, output)
  8. knitr:::process_group.block(group)
  9. knitr:::call_block(x)
     ...
 37. ggplot2 (local) f(l = layers[[i]], d = data[[i]])
 38. l$compute_aesthetics(d, plot)
 39. ggplot2 (local) compute_aesthetics(..., self = self)
 40. base::lapply(aesthetics, eval_tidy, data = data, env = env)
 41. rlang (local) FUN(X[[i]], ...)
Execution halted"

Therefore in the README I set it to eval=FALSE till correcting this error. @olbeimarton , can you correct it? Or @koralgooll ?

**Comment by olbeimarton** [comment]
Hi, I'm not getting this error. However, since updating the column names to snake case the axis and legend titles are:
get(ontology_id_colname) and get(p_value_type_colname). @koralgooll can you please check this? I only replaced the name in SubramanianTest.R (see commit: https://github.com/ELTEbioinformatics/mulea/commit/2da673259fc68a97b36becce855182de38811f9c).
I can manually rename them if you'd like with the ggplot xlab, ylab and labs functions.

**Comment by olbeimarton** [comment]
The other plots correctly display the new snake-case names, except the lollipop plot, which also loses the colour scaling. @koralgooll can you please look into this? Presumably something else needs updating, but I can't find it.

**Comment by barizona** [comment]
Yes, now it runs for me as well. Maybe it was an RStudio error (sometimes when I restart RStudio some errors disappear)? Now the plot looks like this:
![image](https://github.com/ELTEbioinformatics/mulea/assets/14806292/0e31978d-d6f8-4797-a364-122ce267b301)
